forge-gui/res/cardsfolder/upcoming/scheming_aspirant.txt

```diff
@@ -0,0 +1,10 @@
+Name:Scheming Aspirant
+ManaCost:1 B
+Types:Creature Phyrexian Advisor
+PT:1/3
+T:Mode$ Proliferate | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ LoseGain | TriggerDescription$ Whenever you proliferate, ABILITY
```

Comment:
No idea why you would try to split up the TriggerDescription like that

Reply:
ABILITY for Trigger is only needed when handling Trigger with Charm Effects